src/types/settings.ts

```diff
@@ -0,0 +1,13 @@
+import { IThemeChangedMessage } from './query-runner';
+
+export interface ISettingsProps {
+  actions?: {
+    signOut: Function;
+    changeTheme: Function;
+  };
+  intl?: {
+    message: object;
+  };
+  authenticated: boolean;
+  appTheme: IThemeChangedMessage['theme'];
```

Comment:
This works but it creates a dependency with IThemeChangedMessage which can be a source of bugs because the relationship between IThemeChangedMessage and ISettingProps is not obvious.

Can we create a theme enum? And use that instead?